### PR TITLE
Create 2019-07-29-coreweekly-week-27-2019.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 1st to Sunday 07th of July 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week was a very warm week, indeed. Did you find a solution to stay productive in your area?
```
Comment (matks) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: Why the `indeed` ? 😐

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+If week 26 was a small week if we look at the number of pull request, the week 27 is one of the biggest of this year. Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. And, also, a lot of work has been done on modules. Well done!
```
Comment (matks) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]:
```suggestion
If week 26 was a small week considering the numbers of pull requests, week 27 is one of the biggest of this year ! Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. Also, a lot of work has been done on modules. Well done!
```

Actually work on 1.7.7 started way before 😅at the feature freeze it started

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+* [#61](https://github.com/PrestaShop/gamification/pull/61): Fix the deprecated create_function for PHP 7.2, by [@eternoendless](https://github.com/eternoendless)
+* [#67](https://github.com/PrestaShop/gamification/pull/67): Undefined variable when running in fancybox, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#68](https://github.com/PrestaShop/gamification/pull/68): Bump version to 2.3.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#69](https://github.com/PrestaShop/gamification/pull/69): Release 2.3.1, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+
+### Example Module Mailtheme
+
+* [#1](https://github.com/PrestaShop/example_module_mailtheme/pull/1): Update modules after changes from the core, use two different templat…, by [@jolelievre](https://github.com/jolelievre)
```
Comment (matks) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:118]: PR title is too long so it's cut

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+* [#2](https://github.com/PrestaShop/example_module_mailtheme/pull/2): Update min version, by [@jolelievre](https://github.com/jolelievre)
+
+
+### CsaGuzzleBundle
```
Comment (matks) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:122]: Maybe we can explain that this is a fork we created to have a php7.3 compliant version of this lib ?

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: this article will be published on next monday. and, this current week is very warm, indeed

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+### Auto Upgrade
+
+* [#284](https://github.com/PrestaShop/autoupgrade/pull/284): Display a notice if PHP is below v7.1, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#325](https://github.com/PrestaShop/autoupgrade/pull/325): Prepend and optimize autoloader, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#326](https://github.com/PrestaShop/autoupgrade/pull/326): Add PHPDoc & log when module switch the current theme, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#327](https://github.com/PrestaShop/autoupgrade/pull/327): Update logs around zip upgrade, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+* [#328](https://github.com/PrestaShop/autoupgrade/pull/328): Bump version to 4.9.0, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### classic-rocket
```
Comment (matks) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: Do we list classic-rocket notifs now ?

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 3 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:118]: that's fine anyway.

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 5 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: We usually list all public repositories on the org.
classic-rocket is in the org.
maybe it's time to discuss with the author and see if he agrees to move to "friends of presta" org

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 4 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:122]: should we do that in an article ? or, maybe, is it clear enough on the readme.md of our fork?

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 1 above)
Comment (matks) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: Isn't "indeed" supposed to be used when confirming someone's question ?

Like "it has been warm, right ? - yes indeed"

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 5 above)
Comment (ghost) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: Why not ask the author first then?

Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: the target was to add classic-rocket to the project, but that seems to become a difficult option... 
now that friend of presta exists, it should become the place for incubation.

Comment (mickaelandrieu) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: Let's discuss the place of "Friends of Presta" organization next week with @toutantic.

I'm totally open to "welcome" this theme into the organization, and **I'd like to be allowed to share the community resources** into the official documentation.

We have so many useful modules that could be used as documentation...

Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:160]: I agree. useful resources from the community should have a place in the devdoc.

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 2 above)
Comment (PierreRambaud) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: @matks => `of this year!`

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week was a very warm week, indeed. Did you find a solution to stay productive in your area?
+
+If week 26 was a small week considering the numbers of pull requests, week 27 is one of the biggest of this year ! Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. Also, a lot of work has been done on modules. Well done!
```
Comment (PierreRambaud) [inline news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:24]:
```suggestion
If week 26 was a small week considering the numbers of pull requests, week 27 is one of the biggest of this year! Work on 1.7.7 has started on the develop branch, and the work to finish the 1.7.6 has been merged. Also, a lot of work has been done on modules. Well done!
```

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: ok

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md]: done directly in the doc

File: news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md (hunk 6 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-07-29-coreweekly-week-27-2019.md:24]: already done

**Pull request merged**